ci/scripts/cpp_test.sh

```diff
@@ -36,7 +36,7 @@ case "$(uname)" in
     n_jobs=$(sysctl -n hw.ncpu)
     ;;
   *)
-    n_jobs=1
+    n_jobs=${NPROC:-1}
```

Comment:
FYI: We can use the `NUMBER_OF_PROCESSORS` environment variable for `nproc` on Windows.